src/enums/filing-codes.ts

```diff
@@ -4,6 +4,7 @@ export enum FilingCodes {
   ADDRESS_CHANGE_OT = 'OTADD', // Others - Change of Address
   AFFIDAVIT = 'AFDVT',
   ALTERATION = 'ALTER',
+  ALTERATION_BC_TO_ULC = 'NOALU', // More expensive for BC to ULC.
```

Comment:
Please remove or change silly comment 😛

Reply:
I guess I read that wrong, haha.

Instead of hard-coding that this is more expensive (because that could change), how about just saying that it's a different fee for BC to ULC than the others?

Reply:
it's a 1 line change Severin, can we just merge this so I can continue with more important stuff?  If it becomes less expensive than the $100 fee in the future - I'll update it then